doc/web/content/Download/Source.html

```diff
@@ -21,158 +21,33 @@
   <h2>Generic Instructions</h2>
 
   <p>These instructions apply to any UNIX-like system, including
-    GNU/Linux, most BSD variants, Solaris, Cygwin on Windows,
+    GNU/Linux, most BSD variants, Solaris, WSL and Cygwin on Windows,
```

Comment:
Do we want to remove mention of Cygwin here and focus on WSL support?

Reply:
Thanks, I removed the mention of Cygwin.